canonical-web-and-design/jaas-dashboard
# Upgrade to the beta2 version of jujulib.

## Done

This upgrades `@canonical/jujulib` which contains the fix for the unhandled error when unable to connect to a model.

https://github.com/juju/js-libjuju/pull/53

## QA

- Unfortunately it's not always very easy to trigger a failure to connect to a model so just ensure that it works as expected and hopefully it'll resolve the failure to connect errors.

## Details

Fixes #767
